ayaneo-platform.c

```diff
@@ -880,7 +893,7 @@ static void suspend_mode_register_attr(void)
                         break;
                 case air_plus:
                 case slide:
-                        // Not currently working; do not register
+                        ayaneo_led_mc_attrs[0] = &dev_attr_suspend_mode.attr;
```

Comment:
Off mode is still not working properly. Very frequently a resume event will only bring back the right joystick RGB ring. Oem and keep modes work. I think this is because the LED's either stay on and just have to change color (oem/keep with a color set), or they are off either way (keep with color off) when they return. going from ON (running) to OFF (suspended) to ON (resume) causes the left ring to not come back until the color is written again (brightenss or intensity)

We might need to leave this off until we solve this stupid bug. I don't want to create a situation where this is causing bug reports. Alternatively we could only expose oem and keep for the plus for now, but that is less than ideal if someone wants a color while running.

Reply:
Okay, I'll have another go at it soon.
As you mentioned in our chat, maybe doubling up the write on resume will help.

Reply:
I tried doing that a few different ways and nothing seems to make it work 100% of the time. Best I got was every other suspend working. There seems to be some race condition where the writer thread gets called too quickly for the left ring, but adding a delay doesn't solve it either.

Reply:
I was thinking maybe the controller is going to sleep/low power mode when the state is off and the device suspends.
If that's the case then one of the commands being sent in the left packet triggers wake, and maybe needs a delay before the controller starts responding.

Kinda makes sense since all the commands address left first, then right.

We can confirm by swapping left/right and testing - probably easiest to just swap 0x01 and 0x02 in the definitions. 
If after swapping the left ring returns and the right remains off then we just need to figure out which command is triggering wake and add a delay after sending.

Reply:
Should be resolved now. 
For future me: the fix was to add delays before returning from suspend/resume functions to allow the MCU to execute the commands.